recipes/msys2/all/conandata.yml

```diff
@@ -32,3 +32,11 @@ sources:
              "https://sourceforge.net/projects/msys2/files/Base/x86_64/msys2-base-x86_64-20200517.tar.xz",
       ]
       sha256: "c4443113497acb2d2e285d40b929fc55f33f8f669902595ecdf66a655b63dc60"
+  "20201109":
+    - url: ""
+      sha256: ""
```

Comment:
Why did you add two empty entries?

Reply:
To keep the data model consistent and not have to handle that in the recipe. The two empty lines represent the x86 files which are not available for this version.